lib/compress/zstd_lazy.c

```diff
@@ -1202,7 +1165,7 @@ size_t ZSTD_RowFindBestMatch(
 
         /* Cycle through the matches and prefetch */
         for (; (matches > 0) && (nbAttempts > 0); --nbAttempts, matches &= (matches - 1)) {
-            U32 const matchPos = (head + ZSTD_VecMask_next(matches)) & rowMask;
+            U32 const matchPos = (head + ZSTD_countTrailingZeros64(matches)) & rowMask;
```

Comment:
nit: I would keep the `ZSTD_VecMask_next()` function and just have it wrap `ZSTD_countTrailingZeros64()`. The reason is because that function name actually conveys something, and the documentation is useful.